### PR TITLE
add support for targeting 121

### PR DESCRIPTION
**Pull request by cbellini**
Because

- we have experiments that will target features that will be available in Fx 121

This commit

- adds Fx 121 as a target version

### CODE REVIEW

File: experimenter/experimenter/experiments/constants.py [hunk 1]
```diff
@@ -212,7 +212,8 @@ class Application(models.TextChoices):
         APPLICATION_CONFIG_MONITOR_WEB.slug,
         APPLICATION_CONFIG_MONITOR_WEB.name,
     )
-    DEMO_APP = (APPLICATION_CONFIG_DEMO_APP.slug, APPLICATION_CONFIG_DEMO_APP.name)
+    DEMO_APP = (APPLICATION_CONFIG_DEMO_APP.slug,
+                APPLICATION_CONFIG_DEMO_APP.name)
```
Comment (jrbenny35) [inline experimenter/experimenter/experiments/constants.py:216]: Was this change needed? Maybe you were using a formatter or something

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Just 1 question

File: experimenter/experimenter/experiments/constants.py (hunk 1 above)
Comment (cbellini) [reply experimenter/experimenter/experiments/constants.py]: I was using a formatter, yes, and it aligned it to the format used with other lines, but I can revert it to the original formatting.

**Review by brennie — APPROVED** [review]
